### DISCUSSION TITLE
First time training, not sure how to use/verify model

### DISCUSSION THREAD

**Discussion by neural-loop** [coqui-ai/TTS#2500] [General Q&A]
I'm having trouble testing a multi-speaker model that I fine-tuned from the 'tts_models--multilingual--multi-dataset--your_tts' model. Specifically, I'm not sure if I should be using a speaker-embedding layer or d-vectors, and whether to define my own speaker or use a built-in one. Here's what I did so far:

- I looked in the speakers.json for the your-voice model and found a male speaker to try, "male-pt-4".
- In metadata.txt, I specified the audio file, text, and speaker: audio1|a few things that I I want you to know about very much|male-pt-4.
- My model output doesn't have a speakers.json, but it does have a speakers.pth.
- I tried testing the model with the following command: tts --text "Text for TTS" --model_name "output/vits_art-April-11-2023_11+14AM-30109af2/best_model.pth" --config_path "output/vits_art-April-11-2023_11+14AM-30109af2/config.json" --speaker_idx "male-pt-4" --out_path audioout/output.wav, but it seems to be going to the downloader instead of generating audio.

I'd appreciate any guidance on how to properly fine-tune and test my multi-speaker model. Thank you.

```
tts --text "Text for TTS" --model_name "output/vits_art-April-11-2023_11+14AM-30109af2/best_model.pth" --config_path "output/vits_art-April-11-2023_11+14AM-30109af2/config.json" --speaker_idx "male-pt-4" --out_path audioout/output.wav

Traceback (most recent call last):
  File "/home/user/.local/bin/tts", line 8, in <module>
    sys.exit(main())
  File "/home/user/.local/lib/python3.10/site-packages/TTS/bin/synthesize.py", line 318, in main
    model_path, config_path, model_item = manager.download_model(args.model_name)
  File "/home/user/.local/lib/python3.10/site-packages/TTS/utils/manage.py", line 241, in download_model
    model_type, lang, dataset, model = model_name.split("/")
ValueError: not enough values to unpack (expected 4, got 3)

```

**Comment by 78Alpha** [comment]
Sad to come across this and it's closed. The issue is bothersome and still present. It expects a certain path structure but even when you have it, it fails to generate audio. It's like half of the important stuff is lacking documentation.

**Reply by eginhard** [reply]
The above error occurs because `--model_name` was used to specify a path to a model - in this case `--model_path` should be used instead.
